### DISCUSSION TITLE
Sibling Livewires on page with pagination not working correctly and effecting each other

### DISCUSSION THREAD

**Discussion by N30** [livewire/livewire#5504] [7. Bugs (archived)]
So I have multiple livewire on the same page (6)

There are different livewire classes however they are basically an extended class of a Base Livewire class I created where I put all the similar functions.

I simply installed the Pagination per documentation, the livewire pagination trait and not laravels own.

However when I change a page on one page to page two. If I refresh the page, all the boxes will have changed to page two.

The worse part is that it effect ones that have no pagination and makes them load a non existent page that loads empty.

I assumed this is perfectly normal as I hadn't set the keys, however I went and set the keys for each one with a unique key but it did not fix the issue:
```

<div class="row ui-sortable" id="draggableMultiple">
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.resources'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.companies'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.credentials'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )

        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.products'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.resellers'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.services'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.teams'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )
        @livewire('components.card', [
            'livewire'=> ['name'=>'admin.users'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )

         @livewire('components.card', [
            'livewire'=> ['name'=>'admin.orders'],
            'card_classes'=>'recent-order'
        ],key(rand(1,99999)) )

	</div>
```

As I wrote this I had an idea that perhaps the Class they are extending should not be the one with "    use WithPagination; " and the livewire classes each have to declare their own pagination. But that wasn't the case either and made no difference.

 I am wondering if @livewire('component','params', key('key')) is not the same as using :wire:key? I saw somewhere there is also a :key in addition to :wire:key? I am so lost and been searching for hours.

Here is what my classes look like:
```

<?php

namespace App\Http\Livewire\Admin;

use Livewire\Component;
use App\Http\Livewire\Model as Model;
use Livewire\WithPagination;

class Companies extends Model
{
use WithPagination;
}
```

**Comment by joshhanley** [comment]
@N30 the issue possibly is that each set of pagination is using the same query string parameter name "page". But you didn't share how you paginated your data, so I can't be sure.

To change the pagination name, when you call `->paginate()` you need to pass a different name into the third parameter, like `->paginate(10, [*], 'otherPage');`, see here for details https://laravel.com/docs/9.x/pagination#multiple-paginator-instances-per-page

Hope this helps!
